package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-runner",
-  "version": "0.3.9",
+  "version": "0.4.0",
```

Comment:
It changes the public interface in a backwards-compatible manner, so according to semver the minor version ought to be incremented.